platform/src/main/java/org/stellar/anchor/platform/config/PropertySep10Config.java

```diff
@@ -155,14 +164,14 @@ void validateConfig(Errors errors) {
 
   void validateClientAttribution(Errors errors) {
     if (clientAttributionRequired) {
-      if (ListHelper.isEmpty(getClientAttributionAllowList())) {
+      if (ListHelper.isEmpty(getDefaultAllowClientDomain())) {
```

Comment:
What if anchor support only custodial wallets?

Reply:
The validation is only called when the client attribution is used. The custodial wallet does not require client attribution.